### DISCUSSION TITLE
A3 pages in a A4 document

### DISCUSSION THREAD

**Discussion by inesmartinsll** [joaomlourenco/novathesis#511] [Q&A] [answered]
Dear community,

I'm using the NOVA thesis template.
I'm currently trying to put a P&ID in the annex of my thesis. The thing is I want to put it in A3, landscape. Everything is fine with the landscape, I can do it, but with A3... Not so much. This is the code that I currently have:

\newgeometry{paper=a3paper}
\begin{landscape}
    \begin{figure}
    \centering
    \includegraphics[width=1\linewidth]{Chapters/Figures/Images/sec_100.pdf}
    \caption{P\&ID da secção de vapor}
    \label{fig:sec_vap}
\end{figure}
\end{landscape}
\restoregeometry

It gives me a page that is in landscape but not in A3 since when I write \the\pdfpagewidth x \the\pdfpageheight it gives me 597.50787ptx 845.04684pt and not 1190.98pt x 841.89pt as expected, according to ChatGPT. And even when I download the pdf, these pages aren't any different in size to the other pages that are A4...

If you could help out, I would be extremely grateful.

In the pachage.tex, I have the following pachages:
\usepackage{graphicx}
\usepackage{subcaption}
\usepackage{caption}
\usepackage{siunitx}
\usepackage{multirow}
\usepackage{indentfirst}
\usepackage{placeins}
\usepackage{amsmath}
\usepackage{multicol}
**\usepackage{geometry}**
\usepackage{booktabs}
\usepackage{gensymb}
\usepackage{textcomp}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{array}
\usepackage{expl3}
\usepackage{calc}
\usepackage[version=4]{mhchem}
\usepackage{tabularray}
\usepackage[utf8]{inputenc}
**\usepackage{pdflscape}**

**Comment by joaomlourenco** [comment, marked as answer]
Hello @inesmartinsll 
I believe that the template (original, in GitHub) shows how to typeset a page in A3 format. 
Please download the original template and check the given document (PDF).

**Reply by inesmartinsll** [reply]
Hi @joaomlourenco,

Sorry for the delay in my response. I've done what you told me (I opened the template.pdf and searched) and none of the pages seem to be in A3 format. Besides, I've searched for keywords like A4, A3, page, format in the pdf file and I didn't find anything related to turning a A4 page to a A3. 

If, by any chance, I'm searching in the wrong spot, let me know. If not, is there any chance you might help me?

Thank you so much for your help and time.

**Reply by joaomlourenco** [reply]
Open the PDF on the repository:
https://github.com/joaomlourenco/novathesis/blob/main/template.pdf
and go to pages 42 and 43. They are A4 landscape pages.  That's appendix 2.
Open the file `Chapters/appendix2.tex` and you will see in line 17:
```latex
\begin{newpdflayout}{210mm}{297mm}%{420mm}
```
just use
```latex
\begin{newpdflayout}{297mm}{420mm}
```

Did you give a star to this project already?  
Have you contributed with a donation?
Thanks.

**Reply by inesmartinsll** [reply]
Thank you for your help, it's solved.

I've given a star to the project and I will donate!
